charter-template.html

```diff
@@ -173,6 +173,8 @@ <h3 id="out-of-scope">Out of Scope</h3>
 
         <p>More detailed milestones and updated publication schedules are available on the <a href="https://www.w3.org/wiki/[groupname]/PubStatus">group publication status page</a>.</p>
 
+        <p><span class='todo'>Consider adopting a sane testing policy:</span> To promote interoperability, all changes done in specifications have <a href='https://github.com/w3c/testing-how-to/#test-as-you-commit-pointer-event-whatwg-webperf'>tests</a>.</p>
```

Comment:
Shouldn't this be qualified? i.e. doing tests very early (e.g. before any implementation is available) may not be all that sane :)

(I would suggest another term that "sane" here - "healthy", or "proven")

Reply:
sure, we don't want to overspecify it, but then I think the introductory sentence should clarify that this needs to be adapted, not adopted as is.

Reply:
Added "as such" to stay as minimalistic as possible.